README.md

```diff
@@ -22,7 +22,23 @@ implementation 'org.opensearch.client:opensearch-rest-high-level-client:1.1.0'
 
 ## Python
 
-// Coming Soon...
+Testing shows that this is a four (4) line change. Change the `import` statement, and the Python client for ES (Elasticsearch) and OS (OpenSearch).
+
+Changes related to `import`:
+
+```python
+- from elasticsearch import Elasticsearch
++ from opensearchpy import OpenSearch
+```
+
+Changes related to clients and arguments:
+
+```python
++ opensearch_client = Elasticsearch(ES_SERVICE_URI, use_ssl=True)
+- opensearch_client = OpenSearch(OS_SERVICE_URI, use_ssl=True)
+```
+
+See this commit diff as an example change: [python-client-migration/python_client_migration/main.py](https://github.com/aiven/opensearch-migration-examples/commit/58d35ad29ce0ba8337a383fbfe920f1644a80361)
```

Comment:
Following that link, "This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository."
Would https://github.com/aiven/opensearch-migration-examples/pull/8/commits/78de47151ab9138b07dafb0bdfb94fa18c4b5566 (from the current set of commits) be more appropriate?